sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/receipt/_claims_models.py

```diff
@@ -0,0 +1,156 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""Models for application claims."""
+
+from typing import Any, Dict, Optional, Union
+
+
+class LedgerEntryClaim:
+    """
+    LedgerEntryClaim represents an Application Claim derived from ledger entry data.
+    """
+
+    def __init__(
```

Comment:
[`dataclasses`](https://docs.python.org/3/library/dataclasses.html) can be used to avoid some of this boilerplate

As documented, it can generate `__init__` and `__eq__` for you

Reply:
Oh yeah, you're right. Completely forgot that I could use dataclasses and avoid all the boilerplate. Thanks a lot for the suggestion!

Reply:
Addressed